### PR TITLE
unpin django version

### PR DESCRIPTION
**Pull request by yro**
### Description
Upgrade (or rather, allow to inherit from edx platform) django version. <1.9.0

### Testing
- [x] Unit tests

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review: @efischer19 
- [ ] Code review: @robrap 

FYI: @andy-armstrong 

### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -71,3 +71,5 @@ test/selenium/screenshots/*
 
 # logging
 logs/*.log*
+
+edx-ora2/
```
Comment (yro) [inline .gitignore]: will delete for final commit.

**Comment by robrap** [comment]
👍   Hopefully this makes Gemnasium less cranky.  Thanks!
@jibsheet: How long until Gemnasium brings in the latest once this is merged?  Thanks.

File: .gitignore (hunk 1 above)
Comment (cahrens) [inline .gitignore]: As long as this doesn't break anything, sure. 👍

**Comment by robrap** [comment]
@yro: 2 quick things:
1. Why did you need to add ".0" on 1.9?
2. Your description above reads >1.9, but should probably be <1.9.

**Comment by jibsheet** [comment]
@robrap My understanding is that they update on push/merge but I'm not sure how quickly they notice since I don't see a webhook/Service configuration on this repo.

**Comment by robrap** [comment]
Thanks @jibsheet.

**Comment by yro** [comment]
@robrap also there's no reason for the '0', will fix on review.

**Comment by yro** [comment]
@robrap You mean the *description*! Of course, fixing now.

**Comment by yro** [comment]
@jibsheet Gave me the lowdown on checking gemnasium, will merge ticket as-is and update as this resolves (or doesn't).

**Comment by yro** [comment]
Errors still open on Gemnasium, attempting lower bound: 
https://github.com/edx/edx-ora2/pull/967

**Pull request merged**